server/organizations.go

```diff
@@ -56,17 +57,13 @@ func (r *organizationRequest) ValidDefaultRole() error {
 }
 
 type organizationResponse struct {
-	Links       selfLinks `json:"links"`
-	ID          uint64    `json:"id,string"`
-	Name        string    `json:"name"`
-	DefaultRole string    `json:"defaultRole,omitempty"`
+	Links selfLinks `json:"links"`
+	chronograf.Organization
 }
 
 func newOrganizationResponse(o *chronograf.Organization) *organizationResponse {
 	return &organizationResponse{
-		ID:          o.ID,
-		Name:        o.Name,
-		DefaultRole: o.DefaultRole,
+		Organization: *o,
```

Comment:
see above about explicit. this could panic if `o` is nil

Reply:
:+1: